modal/parallel_map.py

```diff
@@ -477,11 +479,23 @@ async def _map_invocation_inputplane(
     # any reason).
     max_inputs_outstanding = MAX_INPUTS_OUTSTANDING_DEFAULT
 
-    # ------------------------------------------------------------
-    # Helper functions
-    # ------------------------------------------------------------
+    # Input plane does not yet return a retry policy. So we set a default.
+    retry_policy = api_pb2.FunctionRetryPolicy(
+        retries=INPUTS_RETRY_MAX_RETRIES,
+        initial_delay_ms=1000,
+        max_delay_ms=1000,
+        backoff_coefficient=1.0,
+    )
```

Comment:
This effectively makes it so that every function has a retry policy right? Most users don't have a retry policy, so this might be unexpected. But I guess we're not giving this to users yet right? Might want to hold off until we have the retry policy sorted out.